files/mods.json

```diff
@@ -1398,6 +1398,31 @@
         "forge_id": "chattriggers",
         "hash": "c0ec16e7a5a105f68fde9fa580dedb4e"
     },
+    {
+        "id": "soopy",
+        "nicknames": [
+            "soopyv2"
+        ],
+        "display": "SoopyV2",
+        "creator": "Soopyboo32",
+        "discordcode": "dfSMq96RSN",
+        "description": "A ChatTriggers module providing many unique and useful features for Hypixel SkyBlock.",
```

Comment:
but its not chattriggers is it

Reply:
> but its not chattriggers is it

it is still a chattriggers module, the only thing the forge mod does is install the module lol

Reply:
the mod description is "Rendering optimisations for the chattriggers module soopyv2"
i dont think it downloads chattriggers into your mods folder right

Reply:
> i dont think it downloads chattriggers into your mods folder right

it does not we'd probably have to find a way to make sure the user has chattriggers already installed/checked 

or we could just not do this at all its up to you and the other admins

Reply:
chattriggers has a warning box when installing. could do the same thing but for soopy saying that chattriggers needs to be installed too